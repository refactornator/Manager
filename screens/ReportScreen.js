import React, { Component } from 'react';

import NotesScreen from './NotesScreen';

export default class ReportScreen extends Component {
  render() {
    const report = this.props.navigation.state.params;

    return (
      <NotesScreen
        {...this.props}
        report={report}
        keyboardVerticalOffset={65}
      />
    );
  }
}
