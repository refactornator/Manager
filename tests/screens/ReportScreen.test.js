import React from 'react';

import { shallow } from 'enzyme';

import ReportScreen from '../../screens/ReportScreen';

const report = {
  key: 1,
  name: 'William'
};

const navigation = { state: { params: report } };

it("renders the report's screen", () => {
  const render = shallow(<ReportScreen {...{ navigation }} />);
  expect(render).toMatchSnapshot();
});
