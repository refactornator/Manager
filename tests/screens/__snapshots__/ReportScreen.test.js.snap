// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the report's screen 1`] = `
<NotesScreen
  keyboardVerticalOffset={65}
  navigation={
    Object {
      "state": Object {
        "params": Object {
          "key": 1,
          "name": "William",
        },
      },
    }
  }
  report={
    Object {
      "key": 1,
      "name": "William",
    }
  }
/>
`;
